io/snmp/snmp.html

```diff
@@ -57,6 +57,58 @@
     });
 </script>
 
+<script type="text/x-red" data-template-name="snmp set">
+    <div class="form-row">
+         <label for="node-input-host"><i class="fa fa-globe"></i> Host</label>
+         <input type="text" id="node-input-host" placeholder="localhost">
+    </div>
+    <div class="form-row">
+        <label for="node-input-community"><i class="fa fa-user"></i> Community</label>
+        <input type="text" id="node-input-community" placeholder="public">
+    </div>
+    <div class="form-row">
+        <label for="node-input-version"><i class="fa fa-bookmark"></i> Version</label>
+        <select type="text" id="node-input-version" style="width: 150px;">
+            <option value="1">v1</option>
+            <option value="2c">v2c</option>
+        </select>
+    </div>
+    <div class="form-row">
```

Comment:
This could be applied to all node (multiple oids seperate by comma ATM). If we get there, snmp-set would add the type and value field, but IMO we have to have, when possible, a similar experience between nodes. I agree that at the moment there is nothing for varbinds in the node.